starfish/core/spacetx_format/test_coordinates.py

```diff
@@ -1,11 +1,15 @@
 import pytest
 from pkg_resources import resource_filename
+from slicedimage import VERSIONS
```

Comment:
I think you need #1334 for this. :(

Looks like travis runs are broken here.

Reply:
:+1: I pip installed locally assuming that'd be the thing to do :wink:  Merged master and pushed.